src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -823,6 +857,45 @@ def test_disp_s1_pge_validate_product_output(self):
             if exists(test_runconfig_path):
                 os.unlink(test_runconfig_path)
 
+    @patch.object(opera.pge.disp_s1.disp_s1_pge.subprocess, "run", mock_grib_to_netcdf)
+    def test_scratch_sas_runconfig_for_grib_to_netcdf_files(self):
+        """
+        Test that the grib_to_netcdf files are in 'disp_s1_pge_test/scratch_dir'.
+        Verify the file names of the .grb files found in ['SAS']['dynamic_ancillary_file_group']['troposphere_files']
+        are the same as the converted files that are now .nc files and are residing in scratch_dir.
+        """
+        starting_grb_file_names = []
+        ending_grb_file_names = []
+        runconfig_path = join(self.data_dir, 'test_disp_s1_config.yaml')
+
+        pge = DispS1Executor(pge_name="DispS1PgeTest", runconfig_path=runconfig_path)
+        # Pull out the starting value of the troposphere_files
+        with open(runconfig_path, 'r', encoding='utf-8') as infile:
+            runconfig_dict = yaml.safe_load(infile)
+        starting_tropo_paths = \
+            runconfig_dict['RunConfig']['Groups']['SAS']['dynamic_ancillary_file_group']['troposphere_files']
+        # Strip the path and extension (.grb) in order to compare just the file names after
+        # conversion and placement of the converted file into /scratch_dir/<fname>.nc
+        for i in starting_tropo_paths:
```

Comment:
Even though this is just test code, I would still like to avoid single-letter loop variables, so could you please rename this to `for starting_tropo_path in starting_tropo_paths`, or similar.